query/query.go

```diff
@@ -112,6 +113,7 @@ var stateName map[byte]string = map[byte]string{
 	15: "orderBy",
 	16: "onDupeKeyUpdate",
 	17: "inNumberInWord",
+	18: "inBackticks",
```

Comment:
In the above definition at line 83 `inBackTicks` = iota + 7 = 0 + 7 = 7.
Is it correct to put it at position 18 here?